app/helpers/navigation_items.rb

```diff
@@ -74,7 +74,15 @@ def for_provider_account_nav(current_provider_user, current_controller, performi
       items = []
 
       unless performing_setup
-        items << NavigationItem.new(t('page_titles.provider.account'), provider_interface_account_path, active?(current_controller, %w[account profile provider_users organisations provider_relationship_permissions]))
+        if FeatureFlag.active?(:accredited_provider_setting_permissions)
+          if current_provider_user.authorisation.can_manage_users_or_organisations_for_at_least_one_setup_provider?
+            items << NavigationItem.new(t('page_titles.provider.organisation_settings'), provider_interface_organisation_settings_path, active?(current_controller, %w[organisations provider_users]))
+          end
+
+          items << NavigationItem.new(t('page_titles.provider.account'), provider_interface_account_path, active?(current_controller, %w[account profile provider_relationship_permissions]))
```

Comment:
Does the `provider_relationship_permissions` controller want to go in the active check for the Org settings nav item?

I think the notifications controller should go in the active check here (which we seem to have missed from before)

Reply:
I guess so. I've added `notifications` to the active check, too.